torch/_torch_docs.py

```diff
@@ -1696,6 +1696,23 @@ def parse_kwargs(desc):
            7.0977)
 """)
 
+add_docstr(torch.get_default_dtype,
+           r"""
+get_default_dtype() -> :class:`torch.dtype`
+
+Get the current default :class:`torch.dtype`.
+
+Example::
+
+    >>> torch.get_default_dtype()  # initial default is float32
+    torch.float32
+    >>> torch.set_default_dtype(torch.double)
+    >>> torch.get_default_dtype()  # default is now changed to float64
+    torch.float64
```

Comment:
Nit: add example with ``torch.set_default_type("torch.DoubleTensor")` so users understand that interaction